### DISCUSSION TITLE
Cannot send performance data

### DISCUSSION THREAD

**Discussion by georgkrause** [getsentry/sentry-python#1563] [Q&A] [answered]
Whenever the sentry sdk tries to send performance data, I get the following:

```
demo-api-1  |  [sentry] ERROR: Unexpected status code: 400 (body: b'{"spans":{"6":{"tags":{"redis.is_cluster":["Not a valid string."]}},"15":{"tags":{"redis.is_cluster":["Not a valid string."]}},"16":{"tags":{"redis.is_cluster":["Not a valid string."]}},"17":{"tags":{"redis.is_cluster":["Not a valid string."]}}}}')
```

Where can I start to find the cause for this? Any help is appreciated! :)

**Comment by sl0thentr0py** [comment, marked as answer]
hey @georgkrause this is actually a bug so moving to issue

**Reply by georgkrause** [reply]
Are you sue, though? Checking the sdk code it seems like any datatype is allowed as tag value, so boolean would be totally legit.

**Reply by sl0thentr0py** [reply]
ok I just tested and boolean actually works. I'm following up on the issue page.
